Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "proton-runner"
+version = "0.0.1"
+authors = ["Ryan Fredlund <fredl027@umn.edu>"]
```

Comment:
Everyone will be adding their own emails as they work on stuff. Consider it a pretest :)